snowpack/src/types.ts

```diff
@@ -353,22 +350,23 @@ export interface PackageSource {
    * for this to complete before continuing. Example: For "local", this involves
    * running esinstall (if needed) to prepare your local dependencies as ESM.
    */
-  prepare(commandOptions: CommandOptions): Promise<ImportMap>;
+  prepare(commandOptions: CommandOptions): Promise<void>;
   /**
    * Load a dependency with the given spec (ex: "/pkg/react" -> "react")
    * If load fails or is unsuccessful, reject the promise.
    */
   load(
     spec: string,
+    isSSR: boolean,
```

Comment:
Same comment as above: is there any danger in changing the param order? It seems like it’d be a better experience to just include this in the `options` object. But please feel free to disagree